tools/import-verifier/import-verifier.go

```diff
@@ -319,6 +319,7 @@ func loadImportRestrictions(configFile string) ([]ImportRestriction, error) {
 func resolvePackage(targetPackage string) ([]Package, error) {
 	cmd := "go"
 	args := []string{"list", "-json", targetPackage}
+	// #nosec
```

Comment:
Please add an explanation about why you are turning off the check here

Reply:
Will do; should I do it as part of the PR description or a comment in the code?

Reply:
done!